<ui version="4.0" >
 <class>GPADialog</class>
 <widget class="QDialog" name="GPADialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Generalised Procrustes Analysis</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Output file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_3" />
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Fitting options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="1" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Alignment</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" >
          <item row="0" column="0" >
           <widget class="QRadioButton" name="radioButton" >
            <property name="text" >
             <string>Align with principal axes</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QRadioButton" name="radioButton_2" >
            <property name="text" >
             <string>Align with first specimen</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QRadioButton" name="radioButton_3" >
            <property name="text" >
             <string>Align with plane of symmetry</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Tangent Projection</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6" >
          <item row="0" column="0" >
           <widget class="QCheckBox" name="checkBox_8" >
            <property name="text" >
             <string>Tangent project coordinates</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Procrustes Fit</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" >
          <item row="0" column="0" >
           <widget class="QCheckBox" name="checkBox_2" >
            <property name="text" >
             <string>Rotate</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="checkBox_3" >
            <property name="text" >
             <string>Translate</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="checkBox" >
            <property name="text" >
             <string>Scale</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QCheckBox" name="checkBox_4" >
            <property name="text" >
             <string>Enable Reflections</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Distances</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Distances between specimens</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7" >
          <item row="1" column="0" colspan="2" >
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="17" column="0" colspan="2" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Note: If you dont know what distances to calculate you should probably
          choose Euclidean distances between all specimens</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QCheckBox" name="checkBox_6" >
            <property name="text" >
             <string>All</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QCheckBox" name="checkBox_7" >
            <property name="text" >
             <string>Pairwise</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Compute distances between
the fitted specimens</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Compare two specimens at a time</string>
            </property>
           </widget>
          </item>
          <item rowspan="3" row="14" column="1" >
           <widget class="QGroupBox" name="groupBox_6" >
            <property name="title" >
             <string>Pairwise</string>
            </property>
            <widget class="QWidget" name="layoutWidget" >
             <property name="geometry" >
              <rect>
               <x>10</x>
               <y>30</y>
               <width>182</width>
               <height>71</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2" >
              <item>
               <widget class="QRadioButton" name="radioButton_4" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string>Procrustes Distances (rho)</string>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_7" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string>Partial Procrustes Distances (Dp)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_6" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string>Full Procrustes Distances (Df)</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item rowspan="3" row="14" column="0" >
           <widget class="QGroupBox" name="groupBox_5" >
            <property name="title" >
             <string>All</string>
            </property>
            <widget class="QWidget" name="layoutWidget_2" >
             <property name="geometry" >
              <rect>
               <x>10</x>
               <y>30</y>
               <width>171</width>
               <height>71</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout" >
              <item>
               <widget class="QRadioButton" name="radioButton_5" >
                <property name="text" >
                 <string>Euclidean Distances</string>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_8" >
                <property name="text" >
                 <string>Tangent Distances</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="checkBox_5" >
            <property name="text" >
             <string>Form Space Distances (include natural log of centroid size)</string>	     
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Algorithm settings</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit" >
       <property name="geometry" >
        <rect>
         <x>150</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2" >
       <property name="geometry" >
        <rect>
         <x>150</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4" >
       <property name="geometry" >
        <rect>
         <x>30</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string>Max iterations</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5" >
       <property name="geometry" >
        <rect>
         <x>30</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string>Min tolerance</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GPADialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GPADialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
